_posts/2018-01-07-building-serverless-radiator.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title: Building a serverless radiator for fun and benefit with Clojure, Python and AWS 
+author: hjhamala
+excerpt: Monitoring software development pipeline statuses, environment alarms and metrics are an essential part of a modern software development process in a cloud environment. In this post I show a simple way to monitor different accounts with a serverless radiator. 
+tags:
+- AWS
+- Clojure
+- Serverless 
+---
+
+***There exists many commercial products for monitoring development pipelines and deployments. What could be more fun than make your own?***
```

Comment:
"There exists" ei paras valinta sanamuodoksi. Ehkäpä jotain tyyliin "Several commercial products are available for monitoring development pipelines and deployments." toimisi paremmin?

Reply:
Myös "making your own".